src/utils/config/template/application.template.entity.ts

```diff
@@ -0,0 +1,24 @@
+import { Field, ObjectType } from '@nestjs/graphql';
+import { QuestionTemplate } from '@utils/config/template/question.template.entity';
+
+@ObjectType()
+export class ApplicationTemplate {
+  @Field(() => String, {
+    nullable: false,
+    description: 'Template application name.',
```

Comment:
Application template name instead of the template application name, if I understand correctly (the name of the application template)

Reply:
Agree.

Reply:
But it was set that way for consistency with the Opportunity template